models/classes/creator/TreeItemLookup.php

```diff
@@ -91,6 +91,10 @@ private function filterTreeData(array $treeData): array
             }
         }
 
+        if (empty($nodeIds)) {
```

Comment:
Could you please apply a similar condition to `ListItemLookup` as well?

```
--- a/models/classes/creator/ListItemLookup.php
+++ b/models/classes/creator/ListItemLookup.php
@@ -66,6 +66,10 @@ class ListItemLookup extends ConfigurableService implements ItemLookup
             $limit
         );
 
+        if (empty($result['nodes'])) {
+            return $result;
+        }
+
         $nodeIds = array_map(
             static function (array $node): string {
                 return $node['uri'];
```

Reply:
it is a bit more rare case. but sure. thanks